qcodes/instrument_drivers/oxford/triton_remote_driver.py

```diff
@@ -0,0 +1,135 @@
+                           )
+
+        self.add_parameter(name='pid_mode',
+                           label='PID mode',
+                           vals=Enum('on', 'off'),
+                           get_cmd='pid_mode?attribute=value',
+                           set_cmd=partial(self.set_with_post, 'pid_mode')
+                           )
+
+        self.add_parameter(name='pid_ramp',
+                           label='PID ramp enabled',
+                           vals=Enum('on', 'off'),
+                           get_cmd='pid_ramp?attribute=value',
+                           set_cmd=partial(self.set_with_post, 'pid_mode')
+                           )
+
+        self.add_parameter(name='pid_setpoint',
+                           label='PID temperature setpoint',
+                           unit='K',
+                           get_parser=float,
+                           get_cmd='pid_setpoint?attribute=value',
+                           set_cmd=partial(self.set_with_post, 'pid_setpoint'))
+
+        self.add_parameter(name='pid_rate',
+                           label='PID ramp rate',
+                           unit='K/min',
+                           get_parser=float,
+                           get_cmd='pid_rate?attribute=value',
+                           set_cmd=partial(self.set_with_post, 'pid_rate'))
+
+        self.add_parameter(name='pid_range',
+                           label='PID heater range',
+                           unit='mA',
+                           get_parser=float,
+                           get_cmd='pid_range?attribute=value',
+                           set_cmd=partial(self.set_with_post, 'pid_range'),
+                           vals=Enum(*self._heater_range_curr))
+
+        for temperature in temperatures:
+            self.add_parameter(temperature,
+                               label=temperature,
+                               unit="K",
+                               get_parser=float,
+                               get_cmd='{}?attribute=value'.format(temperature))
+
+        for pressure in pressures:
+            self.add_parameter(pressure,
+                               label=pressure,
+                               unit="Bar",
+                               get_parser=float,
+                               get_cmd='{}?attribute=value'.format(pressure))
+
+        named_temperatures = ["COOL", 'MC', "PT1", "PT2", "STIL"]
+        for temperature in named_temperatures:
+            self.add_parameter(temperature,
+                               label=temperature,
+                               unit="K",
+                               get_parser=float,
+                               get_cmd='{}?attribute=value'.format(temperature))
+
+    def close(self):
+        self._session.close()
+        super().close()
+
+
+    def ask_raw(self, cmd):
+        response = self._session.get('http://localhost:8000/{}'.format(cmd))
```

Comment:
The address will most likely be the IP of the Triton computer as the remote driver runs on the experiment computer but the server on the triton computer